README.md

```diff
@@ -1,18 +1,21 @@
 <!-- TOC -->
```

Comment:
I think the TOC hasn't been updated correctly. The levels of the different sections do not correspond to the amount of `#` we have.

Reply:
Which ones are wrong? Looks good on my side.

Reply:
I am going to give it a go, I think the levels are not correct.